.github/workflows/black.yml

```diff
@@ -13,4 +13,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: jpetrucciani/black-check@21.12b0
```

Comment:
This change means that we constantly have to update the `black` version in our requirements. Isn't it easier if we just fix a version?

Reply:
Ah, true! I'll change the requirements